doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,9 +63,385 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
+| IP_TYPE | String | Yes | IPV4 or IPV6 |
+| NETWORK_TYPE | String | Yes | VLAN or VXLAN  |
+| TIMEZONE | String | Yes | The timezone where VIM deployed, like UTC+8  |
+| STORAGE_TYPE | String | Yes |  The Storage type, for example "ceph" |
+| HUGEPAGE_ENABLE | String | Yes | TRUE or FALSE  |
+| HUGEPAGE_SIZE | String | Yes | The storage size that hypervisor set for VM, for example "1GB" |
+| QINQ_ENABLE | List | Yes | TRUE or FALSE |
+| HYPERVISOR_CORES | List | Yes | number of CPU bounded for Hypervisor |
+| EXTERNAL_NTP_SERVER_IP | List | Yes | IP list of NTP server, seperated by comma, for example: primariy_IP;second_IP |
+
+<p align="center"><b>Table 6-3-2:</b> Global Settings </p>
+
+
+<a name="6.3.3"></a>
+### 6.3.3 Parameters for network virtualization
+MTU value for network virtualization should be defined, this is usually standard value that defined by user.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NIC_PORT_TYPE | String | Yes | Type for nic port, Manage, Service or Storage |
+| MTU | String | Yes | This describes the row where rack located |
+
+<p align="center"><b>Table 6-3-3:</b> Network Virtualization parameter .</p>
+
+
+<a name="6.3.4"></a>
+### 6.3.4 server information
+Server information should be provided for installer, including full detail info. for each server, nic mapping etc.
+
+#### 6.3.4.1 server information
+First, a table describes the information for each server in the resource pool should be provided.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| NAME | String | Yes | Server name should be aligned with naming rule, this is the unique ID for each device to be referred for identify device |
+| VENDOR | String | Yes | Vendor of device |
+| SKU | String | Yes | The SKU of device, can be referred by other table NIC connection table, to identify slot-port mapping for device  |
+| MODEL | String | Yes | This is the model for different service type, this value is defined from design document of resource pool, example NC1, NC1-S |
+| SN | String | Yes | Serial Number |
+| RES_POOL | String | Yes | Resource pool name |
+| RACK | String | Yes | rack name where device located |
+| POS | String | Yes | the position of device in rack, like 2-3U,4-5U |
+| BMC_IP | String | Yes |  |
+| BMC_GATEWAY | String | Yes |  |
+| BMC_MASK | String | Yes |  |
+| BMC_SUBNET | String | Yes |  |
+| BMC_USR | String | Yes | BMC user |
+| BMC_PWD | String | Yes | BMC password |
+| SERVICE_IP | String | Yes |  |
```

Comment:
It should be an internal IP  configured and used by hardware integration tools, it will be removed after hardware integration verification. 
I have changed it to INTERNAL_IP to make it less confusing.

Reply:
It should be an internal IP  configured and used by hardware integration tools, it will be removed after hardware integration verification. 
I have changed it to INTERNAL_IP to make it less confusing.